ibft/sync/history_test.go

```diff
@@ -135,37 +136,38 @@ func TestFetchDecided(t *testing.T) {
 		{
 			"force error",
 			[]byte{1, 2, 3, 4},
-			[]peer.ID{"2"},
+			[]string{"2"},
 			"2",
 			[]uint64{1, 3, 2},
-			map[peer.ID][]*proto.SignedMessage{},
-			"could not fetch ranged decided instances",
+			map[string][]*proto.SignedMessage{},
+			"could not find highest",
 			errors.New("error"),
 			3,
 		},
 	}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			s := NewHistorySync(test.valdiatorPK, newTestNetwork(t, test.peers, int(test.rangeParams[2]), nil, test.decidedArr, nil), nil, &proto.InstanceParams{
-				ConsensusParams: proto.DefaultConsensusParams(),
-				IbftCommittee:   nodes,
-			}, zap.L())
+			storage := collections.NewIbft(inmem.New(), zap.L(), "attestation")
```

Comment:
same as the comment above - can use badger inmem

Reply:
IMO out of scope for this PR